masu/database/sql/reporting_ocpstoragelineitem_daily_summary.sql

```diff
@@ -0,0 +1,208 @@
+CREATE TEMPORARY TABLE reporting_ocpstoragelineitem_daily_summary_{uuid} AS (
```

Comment:
Since it works, I vote we leave any changes for a SQL optimization issue later. Technically this summary table shouldn't need any GROUP BYs since all the grouping was done in the daily table. It's almost silly to have the summary tables for OCP as they really just do unit conversion. I don't think it's hurting to have this here now, so we can update later.

Reply:
Sounds like a plan to me.